Dockerfile

```diff
@@ -138,6 +143,10 @@ RUN DEBIAN_FRONTEND=noninteractive \
       libstdc++6 \
       zlib1g \
       # end .NET dependencies
+      # charm dependencies
+      ffmpeg \
+      ttyd \
+      vhs \
```

Comment:
I imagine these packages are pretty small? Just resisting making devcontainer much bigger. I think we're fine, but might be nice (add to #5246?) long-term to keep track of our devcontainer size, install time; and in PRs that update it, record how much of an impact it makes. I suspect these deps are fine and we'd be OK to just add a relevant TODO or 2 to that Issue

Reply:
Adding ffmpeg, ttyd, vhs, and expect increased the image size from 4.94 GB to 5.09 GB (~154 MB increase)
will add a TODO

Reply:
is the TODO just to track the dev container size and install time...?